packages/ember/src/-private/request.gts

```diff
@@ -23,6 +23,8 @@ export class Throw extends Component {
   <template></template>
 }
 
+const and = (x: unknown, y: unknown) => x && y;
```

Comment:
and wasn't defined or imported

See also: https://github.com/emberjs/rfcs/issues/816